ide/editor.settings.storage/src/org/netbeans/modules/editor/settings/storage/fontscolors/ColoringStorage.java

```diff
@@ -447,6 +456,12 @@ public ColoringsReader(FileObject f, String mimePath) {
                 attribs = null;
             }
         }
+
+        Color stringToColor(String name) throws Exception {
+            String resolve = colorrefs.getOrDefault(name, name);
+            return ColoringStorage.stringToColor(resolve);
```

Comment:
See previous comment about requiring references to follow a reserved syntax that can't be confused with a real color.

Reply:
Answered there.